app/views/full-page-examples/bank-holidays/index.njk

```diff
@@ -50,9 +50,10 @@ notes: >-
       {% set englandAndWalesHTML %}
 
         {% set englandAndWalesPanelTitleHtml %}
-          <div class='govuk-!-font-size-36 govuk-!-margin-bottom-4'>
+          <span class='govuk-!-font-size-36'>
             The next bank holiday in England and Wales is
-          </div>
+          </span>
+          <br><br>
```

Comment:
Wondering if we could/should just put `display: block` on the span? Not pretty, but neither is random line breaks.